src/screens/index/index.vue

```diff
@@ -8,7 +8,7 @@
           <p>Search the æternity test network blockchain by block, transaction, address. Or go through the last changes or stats.</p>
         </div>
         <div class='search' v-on:keyup.enter='search()'>
-          <input class='search-input' placeholder='Explore Block, Address' v-model='searchString' type="text">
+          <input class='search-input' placeholder='Explore Generation, Block, Address' v-model='searchString' type="text">
```

Comment:
"Block" still should be mentioned in this list? Is it not the same as "generation"?

Reply:
One can search with block-hash, then the block screen will be different from generation and will show only key/micro block data.